File: js/exerciciosLogica/ex08/tabuadaFor.js
let n = parseInt(8);

if (n > 10) {
	console.log("Inválido. Insira um número de 1 a 10 !");
} else {
	for (i = 1; i <= 10; i++) {
		console.log(`${i} x ${n} = ${i * n}`);
	};
};

File: python/tabuadaFor.py
n = int(input("Digite um número de 1 até 10: "))

if n > 10:
    print("Inválido. Insira um número de 1 a 10 !")
else:
    for i in range(0, 11):
        print(f"{i} x {n} = {i * n}")